client/src/ui/organisms/footer/index.scss

```diff
@@ -62,6 +67,10 @@
       margin: 50px 0 30px;
       padding: 30px 0 0;
     }
+
+    a {
+      text-decoration: underline;
+    }
```

Comment:
I believe it is clear that most everything here in the footer are links. For everything to have underlines would look a bit busy(not that that is what this line will do) so, let's go with nothing underlined. The only one that could be a little cryptic is the line that ends with a link to the relevant licenses. But then again, the words "these licenses" clearly suggests to me that those words are a link to the licenses.

Reply:
Done!